### DISCUSSION TITLE
dashboard for total space usage on array from prometheus exporter

### DISCUSSION THREAD

**Discussion by lupasf** [NetApp/harvest#2526] [Q&A]
**Hi lads,**
I'm unable to find a space usage in percentage per array, not per aggregate to import in grafana. Could someone help/advise about this?
What I found is something in cluster, linear graph. Looking for some pie charts, with snapshots, volumes...

**Comment by cgrinds** [comment]
hi @lupasf does the Cluster dashboard have what you're looking for in the `Storage Efficiency Ratios` row? If so, we can explain which metrics those panels use.

<img width="995" alt="image" src="https://github.com/NetApp/harvest/assets/242252/c3f0ffff-2872-40f8-97f8-079bb2093e9b">
